.github/workflows/ci.yml

```diff
@@ -24,7 +22,17 @@ jobs:
       # and hyphens.
       run: echo "TAG=branch-${GITHUB_REF_NAME//[^A-Za-z0-9._-]/-}" | tee -a $GITHUB_ENV
 
-    - run: ./devel/tag $TAG
+    - run: ./devel/build $TAG
+
+    - if: ${{ github.event_name != 'pull_request' }}
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_LOGIN }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - if: startsWith(env.TAG, 'branch-')
+      name: Push $TAG (non-default branch)
+      run: ./devel/push $TAG
```

Comment:
[CI on PR-triggered run failed](https://github.com/nextstrain/docker-base/runs/7983055498?check_suite_focus=true):

> denied: requested access to the resource is denied

Seems transient, [re-running](https://github.com/nextstrain/docker-base/runs/7983378832?check_suite_focus=true).

Reply:
Oops, of course PR-triggered run failed, it's not supposed to attempt push. Will fix.

Reply:
[Fixed](https://github.com/nextstrain/docker-base/compare/f1f9cc6e27bc5ef8ec52300f14bbb29f653470b3..b97688f85d6bb8ebfce3892b6c0922f551348988)